scripts/compose/grafana/grafana.ini

```diff
@@ -0,0 +1,3 @@
+[security]
+admin_user = admin
+admin_password = 1234
```

Comment:
Please add those credentials and the URL where Grafana can be reached also to the service list in the README.